README.md

```diff
@@ -16,6 +16,31 @@ Special thanks to Dominik Walser who was the first developer on this project.
 
 This project uses Django, which is based on python.
 
+## Pre Setup
+
+Before proceeding with the remaining steps, you must ensure that the following Python packages are installed in your environment. If they are not already installed, please install them using pip or your preferred package manager:
+- `Django`
+- `pytz` 
+- `oidc-provider` 
+- `djangorestframework` 
+- `pydantic` 
+- `django-storages` 
+- `django-solo`
+- `django-cors-headers` 
+- `ics` 
+- `cryptography` 
+- `crypto` 
+- `bleach` 
+- `pyotp` 
+- `whitenoise` 
+- `duspython`
+- `weasyprint`
+- `argon2-cffi` 
+
+Make sure to install these packages with compatible versions to avoid dependency issues.
+Example installation command:
```

Comment:
```suggestion
Install the required Python packages:
```

Reply:
If you already have the packages installed, then installing them a second time makes no difference.

Also, we don't need to name the packages twice.